forms.rst

```diff
@@ -702,6 +702,16 @@ the choice is ultimately up to you.
 
         $form->get('dueDate')->setData(new \DateTime());
 
+Rendering the Same Form Multiple Times on the Same Page
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In some situations you might find yourself in need of having the same form on
+the same page multiple times. By default, a form name is based off the class of
+the form type. To differentiate between the forms, you can create a form using
+:method:`FormFactoryInterface::createNamed() <Symfony\\Component\\Form\\FormFactoryInterface>`.
+Doing so, will change the html name of the form and Symfony will be able to see
+which of the forms was submitted, and handle it accordingly.
```

Comment:
I found it confusing, it sounds weird to achieve this, we would need to create different forms in a controller from the same type, pass different views to render them distinctly in a template, without forgetting to handle the request with all the form until one is submitted... very strange!

Why would we need to know which one is submitted if there are the same? I don't get it.

When ONE form from one type needs to be rendered many times, why the name would be different? To handle many rendering using the `form` twig function of one form view, we have to work around the "rendered" attribute, this can be done using:
```twig
{% set rendered_form = form(form) %}

{{ rendered_form }} {# as many times as needed #}
```

To me this is only useful to explicit the name, which is normally generated from the type class name, nothing more.

The only case I see this could be possible is when **the same type is used for many forms but with different options that deeply changes the form's configuration**, and in the same controller. Should we really document such edge cases without making it sound more complex than it really is?

Reply:
@iltar will you have some time to reword this contribution as a short `.. note::` somewhere with the contents proposed by Jules (*"A form name is generated from the type class name, if you want to set it explicitly use ..., you can event suppress it by explicitly set it to an empty string."*)? Thanks!